datafusion/core/tests/sql/select.rs

```diff
@@ -841,7 +841,7 @@ async fn sort_on_window_null_string() -> Result<()> {
     ])
     .unwrap();
 
-    let ctx = SessionContext::with_config(SessionConfig::new().with_target_partitions(2));
```

Comment:
Result of these tests were not deterministic. I have changed so that result would become deterministic.